lib/runner.js

```diff
@@ -53,14 +53,25 @@ Runner.prototype.addSerialTest = function (title, cb) {
 };
 
 Runner.prototype.addBeforeHook = function (title, cb) {
-	this.tests.before.push(new Test(title, cb));
+	var test = new Test(title, cb);
+	test.type = 'hook';
```

Comment:
@sindresorhus yep. May be this is not worth a shot yet, but in future we could want to eliminate these string constants.